id/community/index.md

```diff
@@ -35,7 +35,7 @@ untuk memulai petualangan Anda:
 : Sekarang adalah waktu yang tepat untuk mengikuti perkembangan Ruby.
   Jika berminat untuk membantu, mulailah dari sini.
 
-[Blog tentang Ruby](weblogs/)
+[Ruby Blogs](weblogs/)
```

Comment:
because Bahasa Indonesia does not have plural form, what do you think if we change to "Blog Ruby", "Daftar Blog Ruby", or use the original "Blog tentang Ruby"?

Reply:
"Blog Ruby" sounds good.

By the way, the current title of `id/community/weblogs/index.md` is "Blog". Should I also update this when updating to "Blog Ruby"?

Reply:
@kuntoaji

Reply:
yes I think you should update it too..

Reply:
Updated to "Blog Ruby" @kuntoaji